scripts/content-repo-preview/build.ts

```diff
@@ -0,0 +1,99 @@
+import { promisify } from 'util'
+import { execFileSync } from 'child_process'
+import path from 'path'
+import fs from 'fs'
+
+async function main() {
+  // imports
+  const execFile = promisify((await import('child_process')).execFile)
+
+  // TODO: check for necessary env vars?
+
+  const repo = process.env.REPO
+
+  // our CWD
+  const cwd = process.cwd()
+  const globalCSSFile = path.join(cwd, 'src', 'pages', 'style.css')
+
+  /**
+   * Check for a .next folder, if found copy it back into our website-preview dir
+   * This should allow us to take advantage of Vercel's build cache
+   */
+  if (fs.existsSync(path.join(cwd, '..', '.next'))) {
+    console.log('.next folder found, moving into website-preview...')
+    await execFile('mv', ['../.next', './.next'])
+
+    if (fs.existsSync(path.join(cwd, '.next', 'cache', 'node_modules'))) {
+      console.log('Found cached node_modules, moving...')
+      await execFile('mv', ['./.next/cache/node_modules', './node_modules'])
+    }
+  }
```

Comment:
If cache is found, move it to the correct spots in the nested `website-preview` directory.